app/src/main/java/com/example/harsh/ceefy/LinkedWeb.java

```diff
@@ -240,7 +241,7 @@ protected void onPostExecute(Boolean status){
             }
             if(status){
                 //If everything went Ok, change to another activity.
-                Intent startProfileActivity = new Intent(LinkedWeb.this, HomeActivity.class);
+                Intent startProfileActivity = new Intent(LinkedWeb.this, UpdateProfile.class);
```

Comment:
After linkedIn login, you had functionality in UpdateProfile, hence changed this.